release-notes/release-notes52.md

```diff
@@ -76,7 +76,12 @@ This change applies to the default {{site.productname}} icon pack.
 
 For an updated list of {{site.productname}} icons, see: [Editor icon identifiers]({{ site.baseurl }}/advanced/editor-icon-identifiers/).
 
-### Updated the colorinput component visual appearance --- updated color swatches in input fields
+### New option for setting the default protocol for the Link and Autolink plugins
+
+A new `link_default_protocol` option has been added to the Autolink and Link plugins. This option allows the default protocol to be prepended to links when a protocol has not been supplied by the user.
```

Comment:
This isn't quite accurate, as it's not a new thing, as previously it always prepended `http://` as the protocol. So this setting just lets you change what was getting prepended.